_posts/2018-04-17-devx-at-iw.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: DevEx at Infinity Works
+author: paul_lucas
+image: laptop_desk.jpg
+---
+
+We attended [QCon in London](https://qconlondon.com) last month (which as an aside, I highly recommend if you've not been before) and one of the day's tracks that caught my attention was looking at "DevEx" or "Developer Experience" - the daily life of developers/engineers and how that might be improved through better tooling, process and skills.  It got me thinking - what is the "DevEx" for engineers at Infinity Works and what are the things we can do to nuture it?
+
+> "DevEx is not just about the tools, but the daily experience at work"
+> - QCon Presentation
```

Comment:
Possible to be more specific about the presentation and link to it maybe?

Reply:
Yeah, I've left that deliberately vague as it's from one of our competitors :)